### PR TITLE
kube-ps1 0.7.0

### PR DESCRIPTION
**Pull request by jonmosco**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
YES
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
YES
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
YES
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
YES
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
YES
-----

### CODE REVIEW

File: Formula/kube-ps1.rb [hunk 1]
```diff
@@ -21,7 +21,6 @@ def caveats; <<~EOS
   end
 
   test do
-    kubeon = ". #{opt_share}/kube-ps1.sh && echo $KUBE_PS1_SHELL"
-    assert_equal "zsh", shell_output("zsh -c '#{kubeon}'").chomp
```
Comment (mistydemeo) [inline Formula/kube-ps1.rb]: Are we still able to test the zsh support?

Comment (jonmosco) [reply Formula/kube-ps1.rb]: Yes, I will add that back in, I was having issue with the characters:
```bash
/usr/local/opt/kube-ps1/share/kube-ps1.sh:27: character not in range
```

Line 27:
KUBE_PS1_SYMBOL_DEFAULT=${KUBE_PS1_SYMBOL_DEFAULT:-$'\u2388 '}

Comment (jonmosco) [reply Formula/kube-ps1.rb]: Im stuck on capturing this output in my test.  Outside brew this test runs without fail, and actually ran without fail previously in brew.

Comment (mistydemeo) [reply Formula/kube-ps1.rb]: Hmm, how about setting `ENV["LC_CTYPE"] = "en_CA.UTF-8"` in the test before you try to invoke zsh? That fixes it in `brew test` for me.

Comment (jonmosco) [reply Formula/kube-ps1.rb]: Perfect.  Thank you!

**Comment by mistydemeo** [comment]
Thanks! For future reference, we prefer version bump PRs to be a single commit with the message in the format `formula version`, e.g. `kube-ps1 0.7.0`. I reworded the commit before pushing.

**Pull request closed without merging**